docs/source/operators/scopes.rst

```diff
@@ -0,0 +1,187 @@
+.. _scopes:
+
+Scopes in Jupyter Server
+========================
+
+A scope has a syntax-based design that reveals which resources it provides access to. Resources are
+objects with a type, associated data, relationships to other resources, and a set of methods that
+operate on them.
+
+``<resource>`` in the RBAC scope design refers to the resource name in the Jupyter Server's API
+endpoints in most cases (prefixed with ``/api/``). For instance, ``<resource>`` equal to ``kernels``
+corresponds to Jupyter Server's API endpoints beginning with ``/api/kernels``.
+
+Available scopes
+----------------
+
+The table below lists all available scopes.
+
+.. list-table::
+   :widths: 25 75
+   :header-rows: 1
+
+   * - Scope
+     - Description
+   * - ``self``
+     - Metascope, grants access to user's own resources only.
+   * - ``all``
+     - Metascope, valid for tokens only. Grants access to everything that the token's owning entity can access.
+   * - ``admin:users``
+     - Grants read, write, create and delete access to users and their authentication state but not their tokens.
+   * - ``admin:users:auth_state``
+     - Grants access to users’ authentication state only.
+   * - ``users``
+     - Grants read and write permissions to users' models apart from tokens and authentication state.
+   * - ``read:users``
+     - Read-only access to users' models apart from tokens and authentication state.
+   * - ``read:users:name``
+     - Read-only access to users' names.
+   * - ``read:users:groups``
+     - Read-only access to users' group names.
+   * - ``users:tokens``
+     - Grants read, write, create and delete permissions to users' tokens.
+   * - ``read:users:tokens``
+     - Read-only access to users' tokens.
+   * - ``admin:groups``
+     - Grants read, write, create and delete access to groups.
+   * - ``groups``
+     - Grants read and write permissions to groups, including adding/removing users to/from groups.
+   * - ``read:groups``
+     - Read-only access to groups' models.
+
+Scopes and APIs
+---------------
+
+.. list-table::
+   :widths: 35 10 25 25
+   :header-rows: 1
+
+   * - Path
+     - Operation
+     - Description
+     - Scopes
+   * - ``/api/groups``
```

Comment:
Are the user & group scopes meaningful in jupyter-server? This looks like a copy of the JupyterHub scopes, many of which don't have meaning in Server.

It also looks like the authorization layer maybe doesn't have or need the 'admin' concept, it's a much simpler `[read:]$resource`.